components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -22,7 +22,7 @@ module micro_p3_interface
                             physics_ptend_init
   use physconst,      only: mwdry, cpair, mwh2o, gravit, rair, cpliq, pi, &
                             rh2o, latvap, latice, tmelt, rhoh2o, rairv 
-  use constituents,   only: cnst_add, pcnst, precip_ice_fluxnam, apcnst, bpcnst, pcnst,&
+  use constituents,   only: cnst_add, pcnst, sflxnam, apcnst, bpcnst, pcnst,&
```

Comment:
if you wanted to keep `precip_ice_fluxnam` for other reasons you could use something like `precip_ice_fluxnam->sflxnam` here.